src/components/Dropdown.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { PropTypes } from 'prop-types';
+
+class Dropdown extends React.Component {
+  state = {
+    displayMenu: false
+  };
+
+  showDropdownMenu = (event) => {
+    event.preventDefault();
+    this.setState({ displayMenu: true }, () => {
```

Comment:
All you need is actually just one function:
```suggestion
    this.setState({ displayMenu: !this.state.displayMenu });
```